moveit_core/package.xml

```diff
@@ -47,6 +48,7 @@
   <depend>trajectory_msgs</depend>
   <depend>visualization_msgs</depend>
   <depend>xmlrpcpp</depend>
+  <depend>eigen_conversions</depend>
```

Comment:
`eigen_conversions` still mentioned here.

Reply:
Missed this one. Removed.